distributed/cli/utils.py

```diff
@@ -64,3 +65,49 @@ async def cleanup_and_stop():
 
     for sig in [signal.SIGINT, signal.SIGTERM]:
         old_handlers[sig] = signal.signal(sig, handle_signal)
+
+
+def prepare_dask_ssh_options(entity, opt):
```

Comment:
Oh, I missed that you were mutating the options list here. Mutating globals is generally bad practice, instead I recommend copying (or recreating) the options here. You might make use of `copy.deepcopy`, or there may be a better way.

```python
opt = copy.deepcopy(opt)
# you can now mutate `opt`, as it's a unique copy.
```

Reply:
Thanks @jcrist for your recommendation. I reverted the test changes.